hadoop-hdds/client/src/test/java/org/apache/hadoop/hdds/scm/storage/TestBlockInputStream.java

```diff
@@ -280,35 +284,9 @@ public void testRefreshPipelineFunction() throws Exception {
     }
   }
 
-  @Test
-  public void testGetBlockInfoFailWithIOException() throws Exception {
-    GenericTestUtils.setLogLevel(BlockInputStream.getLog(), Level.DEBUG);
-    GenericTestUtils.LogCapturer logCapturer =
-        GenericTestUtils.LogCapturer.captureLogs(
-            LoggerFactory.getLogger(BlockInputStream.class));
-    BlockID blockID = new BlockID(new ContainerBlockID(1, 1));
-    AtomicBoolean isRefreshed = new AtomicBoolean();
-    createChunkList(5);
-    BlockInputStream blockInputStreamWithRetry =
-        new DummyBlockInputStreamWithRetry(blockID, blockSize,
-            MockPipeline.createSingleNodePipeline(), null,
-            false, null, chunks, chunkDataMap, isRefreshed,
-            new IOException("unavailable"));
-    try {
-      Assert.assertFalse(isRefreshed.get());
-      byte[] b = new byte[200];
-      blockInputStreamWithRetry.read(b, 0, 200);
-      // As in case of IOException we do not do do refresh.
-      Assert.assertFalse(isRefreshed.get());
-      Assert.assertTrue(logCapturer.getOutput().contains(
-          "Retry to get chunk info fail"));
```

Comment:
This is the only test that excercises the log, and it's been removed.  Is that a problem?

Reply:
That test was to confirm the exception handling for IOException, which is now tested in the parameterized `testReadNotRetriedOnOtherException` and `testRefreshOnReadFailure`.